test/functional/localConfig.json

```diff
@@ -4,6 +4,12 @@
       "browser": [
         "all"
       ],
+      "saucelabs": [
```

Comment:
Saucelabs environment is read from this configuration file by the Testcafe testrunner.js